### ISSUE TITLE
Render option to skip automatic tlmgr updates

### ISSUE THREAD

**Issue by jdutant** [quarto-dev/quarto-cli#11531]
**Problem**

When rendering to PDF, simple LaTeX typos trigger an automatic tlmgr update. These can be slow, especially if you're not connected to the internet. These automatic updates are frustrating when I know that I'm not trying to use fancy LaTeX packages, just making typos.

**MWE**

``` markdown
Silly LaTeX mistake: $P & Q$.
```

Result of `quarto render -t pdf`:

```
Rendering PDF
running xelatex - 1
  This is XeTeX, Version 3.141592653-2.6-0.999996 (TeX Live 2024) (preloaded format=xelatex)
   restricted \write18 enabled.
  entering extended mode
  
updating tlmgr

updating existing packages
ERROR: 
compilation failed- error
Misplaced alignment tab character &.
l.160 Silly LaTeX mistake: \(P &
                                 Q\). 
```

If not connected to the internet, the `updating tlmgr` stage lasts for a good while.

**Proposed solution**

It'd be very useful to have a `--no-tlmgr` option on the command line!

**Comment by cscheid** [comment]
I 100% agree, and this annoys me to no end as well!

**Comment by cderv** [comment]
We have already a configuration to opt out auto installing (https://quarto.org/docs/output-formats/pdf-engine.html#package-installation) 

If you use 
````yaml
latex-auto-install: false
````

this should not try to update tlmgr and install any missing package and so try to solve any error from the log by updating and downloading missing package. 

Did you try that already ? 

By opting out auto install, only feature left by the Quarto engine is the multiple rendering from bibliography and   index. 

To opt out completely of Quarto PDF engine logic, and use LaTeX directly, this option can be set 
````yaml
latex-auto-mk: false
````

Then `latexmk` pdf engine could be used directly for example like so https://quarto.org/docs/output-formats/pdf-engine.html#latexmk

Anyhow, I do think that `latex-auto-install` configuration is doing that already. It is not a CLI flag though. It needs to be passed as metadata. 

Hope it helps

**Comment by cscheid** [comment]
> It is not a CLI flag though. It needs to be passed as metadata.

`quarto render -M latex-auto-mk:false` should work, though!

**Comment by jdutant** [comment]
Thanks! My bad, I hadn't seen that option.

**Comment by cderv** [comment]
> > It is not a CLI flag though. It needs to be passed as metadata.
> 
> `quarto render -M latex-auto-mk:false` should work, though!

yes that is what I meant - should have shared an example 😅 Thanks !
